src/components/axes/timeAxis.ts

```diff
@@ -207,6 +202,19 @@ export module Axis {
         return this._possibleTimeAxisConfigurations;
       }
       this._possibleTimeAxisConfigurations = configurations;
+      this._numTiers = _Util.Methods.max(this._possibleTimeAxisConfigurations.map((config: TimeAxisConfiguration) => config.length), 0);
+
+      if (this._isAnchored) {
+        this._createDomElements();
+      }
```

Comment:
I was thinking of that, but the problem is the following: In order to compute the layout, we need the size of the tiers. But in order to know the size of the tiers, we need to first create the DOM elements, put them at the right position, and measure them (then recycle them). This is what I understood from a discussion with @jtlan . Please correct me if I am wrong